butler.py

```diff
@@ -259,6 +259,8 @@ def main():
       'package', help='Package clusterfuzz with a staging revision')
   parser_package.add_argument(
       '-p', '--platform', choices=['linux', 'macos', 'windows', 'all'])
+  parser_package.add_argument(
```

Comment:
Can we use a different name than "stage" which is used to mean the appengine stuff.
Why do you think it's worth using this?

Reply:
No reason in particular, it is the first word that came to my mind to represent this concept.
To have a more clear meaning, this could be a `release`, for which we would have prod and candidate. Wdyt?